2020/day3/solution.py

```diff
@@ -0,0 +1,43 @@
+file = open('input.txt')
+
+class grid:
+    
+    def __init__(self, lines):
+        self.lines = list(lines)
+
+    def isTree(self, x, y):
+        if y > self.height():
+            return False
+        row = self.lines[self.height() - y]
```

Comment:
```suggestion
        row = self.lines[-y]
```
Negative slice indexes are valid and handy!

Reply:
oh, i knew you could do them on  slices, didn't realize you could do  indices as well.  that  is handy